CvGameCoreDLL_Expansion2/CvCity.cpp

```diff
@@ -6743,7 +6743,7 @@ void CvCity::DoEventChoice(CityEventChoiceTypes eEventChoice, CityEventTypes eCi
 						if (pkEventChoiceInfo->IsApplyEffectToSpyOwner())
 						{
 							if (GET_PLAYER(eSpyOwner).getCapitalCity() != NULL)
-								GET_PLAYER(eSpyOwner).getCapitalCity()->DoEventChoice(eEventChoice, NO_EVENT_CITY, true, -1, getOwner());
+								GET_PLAYER(eSpyOwner).getCapitalCity()->DoEventChoice(eEventChoice, NO_EVENT_CITY, false, -1, getOwner());
```

Comment:
I also changed this one `bSendMsg` to `false` - it is not tied with issue but looks like we don't need to send network events here too. We should send network just when human is pressing some button but not inside some logic which already fires on every end.